CMakeLists.txt

```diff
@@ -77,18 +80,8 @@ set(EP_BASE ${CMAKE_BINARY_DIR}/ThirdParty)
 # Determine common toolchain settings, compiler and link flags
 include(CommonCompilerConfig)
 
-# Set up external dependencies.
-#
-# This will try find_library and find_program and if that fails use
-# ExternalProject to fetch and build the libraries.
-#
-# Sets the following variables:
-#   CAPNP_LINK_LIBRARIES
-#   EXTERNAL_INCLUDE_DIRS
-#
-# Defines the follow functions:
-#   create_capnpc_target - Creates target that generates C++ files from
-#       .capnp schema files.
+# Set up builds of external dependencies and get get their exported settings.
+# (see individual external/*.cmake modules for exported settings)
```

Comment:
@scottpurdy: is this reference "see individual external/*.cmake modules for exported settings" a reasonable substitute? It certainly won't get out of sync with any of the modules. I think it may be better than information that's out of sync with what's actually in the external modules (?).